docs/variants/pc_engines/firmware-update.md

```diff
@@ -0,0 +1,42 @@
+# Firmware update
+
+The following documentation describes the process of Dasharo open-source
+firmware update. If your device is currently flashed with the proprietary
+firmware please refer to the [Initial deployment](initial-deployment.md)
+documentation.
+
+The update process may be different, depending on the currently installed
+Dasharo firmware version.
+
+For simplicity of the update process, we recommend using
+[Dasharo Tools Suite](../../dasharo-tools-suite/overview.md).
+
+Before updating, ensure the SPI flash WP pin is not active (BISO write protect
+pin jumper on J3).
```

Comment:
Is it the same for all versions of PC Engines?

Reply:
Hmm on apu2 it is J2, the rest is J3... Good catch

Reply:
https://github.com/Dasharo/docs/pull/794/commits/d737a9610ee8cc3237969a7fe7bba2948700009b